examples/example-fastmcp-mcp/README.md

```diff
@@ -0,0 +1,259 @@
+```
+
+### Step 4: Configure the API and Default Audience
+
+This step creates the API (also known as a Resource Server) that represents your protected MCP tools and sets it as the
+default for your tenant.
+
+1. Create the API: This command registers the API with Auth0, defines its signing algorithm, enables Role-Based Access
+   Control (RBAC), and specifies the available scopes. Replace `http://localhost:3001` and `MCP Tools API`
+   with your desired identifier and name. Add your tool-specific scopes to the scopes array.
+
+   Note that `rfc9068_profile_authz` is used instead of `rfc9068_profile` as the token dialect to enable RBAC. [Learn more](https://auth0.com/docs/get-started/apis/enable-role-based-access-control-for-apis#token-dialect-options)
+
+```
+auth0 api post resource-servers --data '{
+  "identifier": "http://localhost:3001",
+  "name": "MCP Tools API",
+  "signing_alg": "RS256",
+  "token_dialect": "rfc9068_profile_authz",
+  "enforce_policies": true,
+  "scopes": [
+    {"value": "tool:whoami", "description": "Access the WhoAmI tool"},
+    {"value": "tool:greet", "description": "Access the Greeting tool"}
+  ]
+}'
+
+```
+
+2. Set the Default Audience: This ensures that users logging in interactively get access tokens that are valid for your
+   newly created MCP API. Replace `http://localhost:3001` with the same API identifier you used above.
+
+```
+auth0 api patch "tenants/settings" --data '{"default_audience": "http://localhost:3001"}'
+```
+
+### Step 5: Configure RBAC Roles and Permissions
+
+Now, set up roles and assign permissions to them. This allows you to control which users can access which tools.
+
+1. Create Roles: For each role you need (e.g., "Tool Administrator", "Tool User"), run the create command.
+
+```
+# Example for an admin role
+auth0 roles create --name "Tool Administrator" --description "Grants access to all MCP tools"
+
+# Example for a basic user role
+auth0 roles create --name "Tool User" --description "Grants access to basic MCP tools"
+```
+
+2. Assign Permissions to Roles: After creating roles, note the ID from the output (e.g. `rol_`) and and assign the API
+   permissions to it. Replace `YOUR_ROLE_ID`, `http://localhost:3001`, and the list of scopes.
+
+```
+# Example for admin role (all scopes)
+auth0 roles permissions add YOUR_ADMIN_ROLE_ID --api-id "http://localhost:3001" --permissions "tool:whoami,tool:greet"
+
+# Example for user role (one scope)
+auth0 roles permissions add YOUR_USER_ROLE_ID --api-id "http://localhost:3001" --permissions "tool:whoami"
+```
+
+3. Assign Roles to Users: Find users and assign them to the roles.
+
+```
+# Find a user's ID
+auth0 users search --query "email:\"example@google.com\""
+
+# Assign the role using the user's ID and the role's ID
+auth0 users roles assign "auth0|USER_ID_HERE" --roles "YOUR_ROLE_ID_HERE"
+```
+
+### Step 6: Configure and Deploy the Scope-Injection Action
```

Comment:
Is this section absolutely needed? I just went ahead and skipped it and my AT contains the `scope` claim with the right value (derived from the RBAC roles + permissions defined in Step 5).
Can we double-check that this is indeed needed or perhaps isn't needed anymore?

We could leave a comment about the fact that any further customization not supported out of the box by RBAC _can_ be done via a custom Post-Login action trigger, but I'd consider that optional.